sdk/python/feast/repo_config.py

```diff
@@ -130,6 +130,8 @@ class RegistryConfig(FeastBaseModel):
     sqlalchemy_config_kwargs: Dict[str, Any] = {}
     """ Dict[str, Any]: Extra arguments to pass to SQLAlchemy.create_engine. """
 
+    cache_mode: StrictStr = "sync"
```

Comment:
Please write the docstring for this parameter. So all users know what modes are available.